nerdlandbot/commands/GuildData.py

```diff
@@ -242,6 +245,26 @@ async def remove_purger(self, text_channel) -> bool:
             # purger text channel does not exist in list, return False
             return False
 
+    async def update_notification_audit(self, list_name: str) -> bool:
+        """
+        Updates the notified_on field for the notified list
+        :param list_name: The list that needs to be updated (str)
+        :return: True if updated successfully, False if something went wrong. (bool)
+        """
+        if not list_name in self.notification_lists.keys():
+            return False
+
+        notification_list = self.notification_lists[list_name]
+        
+        if not "notified_on" in notification_list.keys():
+            notification_list["notified_on"] = []
+
+        notification_list["notified_on"].append(datetime.now().isoformat())
+        await self.save()
+
+        return True
```

Comment:
If you are not going to do anything with the return value, why even return a value at all?

Reply:
still useful for future handling. Would keep is kind of a good practice.